tests/kernel/sched/schedule_api/testcase.yaml

```diff
@@ -4,18 +4,22 @@ tests:
     extra_configs:
       - CONFIG_TIMESLICING=y
     tags: kernel threads sched userspace
+    min_ram: 36
   kernel.scheduler.no_timeslicing:
     filter: not CONFIG_SCHED_MULTIQ
     extra_configs:
       - CONFIG_TIMESLICING=n
     tags: kernel threads sched userspace
+    min_ram: 36
```

Comment:
how did you come up with this number of 36?

Reply:
the failure board has RAM=32, so I just enlage it to 36 to skip those failure boards.

Reply:
well, we should not exclude tests from majority of boards based on 1 single board constraints. Overflows right now are being skipped in sanitycheck and should not be considered as failures, if a test does not fit in a board, it is not a bug.
Where are you getting those failures?

Reply:
well, when I manuall check the kernel cases, I guess there shall be some clear explaination on why this case is skipped.